autoload/fzf/vim.vim

```diff
@@ -615,7 +626,7 @@ function! fzf#vim#gitfiles(args, ...)
   endif
   if a:args != '?'
     return s:fzf('gfiles', {
-    \ 'source':  'git ls-files '.a:args.(s:is_win ? '' : ' | uniq'),
+    \ 'source':  'git -C ' . root . ' ls-files '.a:args.(s:is_win ? '' : ' | uniq'),
```

Comment:
We need to `shellescape` root. i.e. `fzf#shellescape(root)`
To avoid repetition, I suggest defining a variable and reusing it throughout.

```vim
let prefix = 'git -C ' . fzf#shellescape(root) . ' '
```

Reply:
Fixed. Also fixed `s:commits()`.